lib/modules/apostrophe-assets/index.js

```diff
@@ -575,6 +575,17 @@ module.exports = {
         var last = chain[chain.length - 1];
         var from = self.getAssetRoot() + '/public/modules/' + name;
         var to = last.dirname + '/public';
+        var scope = to.match(/@.+?(?=\/)/);
```

Comment:
Maybe search for a match in the `name` variable instead? I think this could be faster ...

Reply:
Speed isn't much of an issue in this method which runs once at startup, but I think matching on `name` would be more correct, as it avoids any issues with atypical folders with `@` in them earlier.